### PR TITLE
Store the dqm file

### PR DESCRIPTION
**Pull request by apsallid**
An option is added (`--keepDQMfile`) so that, if used, at the RECO step the DQM file will also be stored locally or in eos in a separate folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,11 +95,13 @@ python SubmitHGCalPGun.py \
   --queue QUEUENAME \
   --inDir partGun_[MYTAG]_[DATE] \
   [--local] \
-  --tag MYTAG
+  --tag MYTAG \
+  [--keepDQMfile]
```
Comment (rovere) [inline README.md]: I'd remove the square brackets since this option has a default

File: SubmitHGCalPGun.py [hunk 2]
```diff
@@ -309,11 +313,13 @@ def submitHGCalProduction():
 
             cfgfile = basename +'.py'
             outfile = basename +'.root'
+            outdqmfile = basename.replace(opt.DTIER, 'DQM') +'.root'
```
Comment (rovere) [inline SubmitHGCalPGun.py]: It'd be nice to have a template name that, if uploaded to a private DQM GUI, is silently digested correctly. This naming will miserably fail.

Comment (apsallid) [reply SubmitHGCalPGun.py]: Hi @rovere , Do you have any twiki with guidelines so that to have a properly name file?

Comment (rovere) [reply SubmitHGCalPGun.py]: uhmm.
Here https://github.com/rovere/dqmgui/blob/index128/src/python/DQM/visDQMUtils.py
you can find how the name is inspected and decomposed. But I realize it's not immediate to understand.
Let me think about it a little longer and I can propose something.

**Comment by apsallid** [comment]
Hi @rovere , @clelange , 

I just wanted to clarify that the DQM file that is stored with this PR is in edm format. The harvesting step is not executed in order to save the DQM file in plain ROOT format. 
I thought in this way this would be more generic, but if you feel like we should add the harvesting step I can work on that. However, that may be useful for me, but for others it may add more unnecessary complexity to the repository. Personally, I just add an extra cmsDriver command in the RECO step in the SubmitFileGSD.sh script, which of course is not the correct way to do since all the options are hardcoded, but it does the job.

**Comment by clelange** [comment]
@apsallid - I think we should just use whatever works best for you, you're the main user of this feature. If anyone else wants to use this, we can re-discuss. 
@rovere - Are there any short-term plans from your side or anyone else you could think of who might want to use anything beyond what Andreas has implemented

**Pull request merged**